### PR TITLE
RoR Group Capstone Project: Recipe App

### PR DESCRIPTION
**Pull request by dazehinn**
### Features
- created login page and registration page with Devise.
- Food list displays a list of food added by the logged-in user (display also quantity of a given food).
- created a form that allows users to add new food.
- displays a list of recipes created by the logged-in user which leads to recipe details.
- user can delete a recipe
- displays a list of all public recipes which leads to recipe details.
- recipe details displays a toggle button that allows  a recipe to be made public or private.
- Generates shopping list view
- displays the list of food that is missing for all recipes of the logged-in user
- count the total food items and total price of the missing food.
- a navigation menu that allows users to open all  created pages
- unit and integration tests
- fix N+1 queries

Before
before

After
after

### CODE REVIEW

File: spec/views/food_index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    it 'Shows the price of each food' do
+      expect(rendered).to have_content '$4.00'
+      expect(rendered).to have_content '$3.00'
+    end
+
+    it 'Shows a delete button to delete a food' do
+      expect(rendered).to have_link('Delete')
+    end
+  end
```
Comment (shella12) [inline spec/views/food_index_spec.rb:28]: - [ ] Great job on creating test cases for your project using `rspec` however according to the [project requirements](Write integration tests with capybara gem) you should write your integration tests as well. Please use the required integration tests for your application by using `capybara`.

Project Requirements:
----
Technical setup

> - Set up RSpec and Capybara testing libraries.
> - Write integration tests with capybara gem

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @dazehinn and @oovillagran , 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- All tests passed ✅ 
- App UI matches wireframe ✅  
- Working app ✅   
- Fixed N+1 problem ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lizdev-05 — APPROVED** [review]
Hi @dazehinn and @oovillagran , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

:heavy_check_mark: Professional Readme file :+1: 
:heavy_check_mark: Good commit history :+1:
:heavy_check_mark: No linter error :+1: 
:heavy_check_mark: All tests pass :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by oovillagran** [comment]
Thanks @Lizdev-05! 👌

**Pull request merged**